dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -2317,6 +2316,28 @@ Spdp::Interceptor::reactor_is_shut_down() const {
   return task_->is_shut_down();
 }
 
+void
+Spdp::schedule_send(const ACE_Time_Value& delay)
```

Comment:
Could you use `TimeDuration` for this? I would like use to not use `ACE_Time_Value` directly in the core libraries unless interacting with ACE.

Reply:
Absolutely.